tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "ignoreDeprecations": "5.0",
```

Comment:
I plan on tackling this in another PR, since that'll involve quite a bit of work